rabbit_mq/variables.tf

```diff
@@ -0,0 +1,109 @@
+variable "region" {
+  description = "AWS Default Region"
+  default     = "us-east-1"
+}
+
+variable "profile" {
+  type        = string
+  default     = "default"
+  description = "Credentials profile for AWS provider"
+}
+
+variable "user_tags" {
+  description = "Mandatory tags for the RabbitMQ resources"
+  type        = map(string)
+}
+
+variable "octopus_tags" {
+  description = "Octopus Tags"
+  type        = map(string)
+}
+
+variable "broker_name" {
+  type        = string
+  description = "Name of the Amazon MQ Broker"
+}
+
+variable "engine_type" {
+  default     = "RabbitMQ"
+  description = "This terraform module will currently support only RabbitMQ"
+}
+
+variable "engine_version" {
+  type        = string
+  default     = "3.8.11"
+  description = "This is the engine version for RabbitMQ"
+  validation {
+    condition     = contains(["3.8.6", "3.8.11"], lower(var.engine_version))
+    error_message = "Unsupported engine version for AWS MQ where engine type is RabbitMQ."
+  }
+}
+
+variable "broker_instance_type" {
+  type        = string
+  default     = "mq.m5.large"
+  description = "This is the broker instance type for RabbitMQ"
+  validation {
+    condition     = contains(["mq.m5.large", "mq.m5.xlarge", "mq.m5.2xlarge", "mq.m5.4xlarge"], lower(var.broker_instance_type))
+    error_message = "Unsupported broker instance type for AWS MQ where engine type is RabbitMQ."
+  }
+}
+
+variable "username" {
+  type        = string
+  default     = "rmqadmin"
+  description = "Admin username for AWS MQ where engine type is RabbitMQ"
+}
+
+variable "auto_minor_version_upgrade" {
+  type        = bool
+  default     = false
```

Comment:
Make it true by default. Make it easier for us to manage, unless otherwise specified.

Reply:
done